src/conda_patches/0003-Return-unknown-module-in-deprecations.patch

```diff
@@ -1,19 +1,23 @@
 From: jaimergp
-Date: Wed Jun 21 18:53:36 CEST 2023
+Date: Tue Feb 27 12:01:36 CEST 2024
 Subject: Return 'unknown' instead of raising an exception when the calling module cannot be determined
 
 diff --git a/conda/deprecations.py b/conda/deprecations.py
-index 61154a1ee..9273cdde6 100644
+index 5d4c0df28..924428fa0 100644
 --- a/conda/deprecations.py
 +++ b/conda/deprecations.py
-@@ -284,7 +284,9 @@ class DeprecationHandler:
-             module = inspect.getmodule(frame[0])
-             return (module, module.__name__)
-         except (IndexError, AttributeError):
--            raise DeprecatedError("unable to determine the calling module") from None
-+            # raise DeprecatedError("unable to determine the calling module") from None
-+            import sys as unknown
-+            return (unknown, "<unknown>")
```

Comment:
I'm not following, why is this needed?

Reply:
PyInstaller will fail to determine the module being deprecated because some attribute was missing during the inspection. It was introduced in https://github.com/conda/conda-standalone/pull/25. I'll comment it out to see what happens (maybe we don't need it anymore?), and worst case I can post the traceback for posterity.